src/component/parser/json/converter/DiagramConverter.ts

```diff
@@ -116,11 +117,16 @@ export default class DiagramConverter extends AbstractConverter<BpmnModel> {
         }
       }
 
-      const label = this.deserializeLabel(shape.BPMNLabel, shape.id);
+      const label = this.getLabelForShape(shape, bpmnElement, bounds);
       return new Shape(shape.id, bpmnElement, bounds, label, shape.isExpanded);
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private getLabelForShape(shape: any, bpmnElement: ShapeBpmnElement, bounds: Bounds): Label {
+    return bpmnElement.kind === ShapeBpmnElementKind.TEXT_ANNOTATION ? new Label(new Font(), bounds) : this.deserializeLabel(shape.BPMNLabel, shape.id);
```

Comment:
Why did you do that ?
A Shape of a textAnnotation can have a BPMNLabel & a Font.

**Example from B.2.0**:
```
<semantic:definitions id="_1373638079286" name="B.2.0" targetNamespace="http://www.trisotech.com/definitions/_1373638079286" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:bpsim="http://www.bpsim.org/schemas/1.0" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:semantic="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">   
   <semantic:process isExecutable="false" id="Process_ba16239e-181e-4b9f-bc5b-0bb2ee973450">          
      <semantic:textAnnotation id="_4815ea6a-ede2-489b-8b37-2cdb2835b02c">
         <semantic:text>Annotation</semantic:text>
      </semantic:textAnnotation>
      <semantic:association associationDirection="None" sourceRef="_303e68ec-dbb3-4d90-8a96-26e0be44f5f3" targetRef="_4815ea6a-ede2-489b-8b37-2cdb2835b02c" id="_5362a7ef-ce7e-4a91-9c38-66c07b1b5f49"/>
   </semantic:process>
<bpmndi:BPMNDiagram documentation="" id="Trisotech_Visio-_6" name="B.2.0" resolution="96.00000267028808">
      <bpmndi:BPMNPlane bpmnElement="C1373638080953">
         <bpmndi:BPMNShape bpmnElement="_4815ea6a-ede2-489b-8b37-2cdb2835b02c" id="S1373638080887__4815ea6a-ede2-489b-8b37-2cdb2835b02c">
            <dc:Bounds height="23.0" width="102.0" x="930.0" y="344.0"/>
            <bpmndi:BPMNLabel labelStyle="LS1373638080849"/>
         </bpmndi:BPMNShape>
      </bpmndi:BPMNPlane>
      <bpmndi:BPMNLabelStyle id="LS1373638080849">
         <dc:Font isBold="false" isItalic="false" isStrikeThrough="false" isUnderline="false" name="Arial" size="11.0"/>
      </bpmndi:BPMNLabelStyle>
   </bpmndi:BPMNDiagram>
</semantic:definitions>
```

Reply:
good point,
I have missed that